test/multiverse/suites/concurrent_ruby/concurrent_ruby_instrumentation_test.rb

```diff
@@ -0,0 +1,81 @@
+# This file is distributed under New Relic's license terms.
+# See https://github.com/newrelic/newrelic-ruby-agent/blob/main/LICENSE for complete details.
+# frozen_string_literal: true
+
+class ConcurrentRubyInstrumentationTest < Minitest::Test
+  EXPECTED_SEGMENTS_FOR_NESTED_CALLS = [
+    'Concurrent::ThreadPoolExecutor#post',
+    'Concurrent/Task',
```

Comment:
@tannalynn  - Next time we chat, let's think about whether we need both of these segments or if we can condense them into one.

Reply:
We decided we just need one of them, so we removed `Concurrent::ThreadPoolExecutor#post`.